dark/models/tournament/team/role/role.py

```diff
@@ -11,6 +11,3 @@ class TeamRole(models.Model):
 
     def __str__(self):
         return self.name
-
-    class Meta:
```

Comment:
Uzasadnienie pls :) - FYI było to po to by w 1 teamie nie powtarzały się nazwy roli

Reply:
Dobrze zgadnąłeś, usunęłem unique'a bo dostawałem błąd po wprowadzaniu niektórych zmian, ale już poprawiłem ten kod.